pkg/agent/controller/agent.go

```diff
@@ -78,24 +77,28 @@ func (a *Controller) Run(stopCh <-chan struct{}) error {
 	return nil
 }
 
-func (a *Controller) serviceExportToRemoteMcs(obj runtime.Object) runtime.Object {
+func (a *Controller) serviceExportToRemoteServiceImport(obj runtime.Object) (runtime.Object, bool) {
 	svcExport := obj.(*lighthousev2a1.ServiceExport)
 	svc, err := a.kubeClientSet.CoreV1().Services(svcExport.Namespace).Get(svcExport.Name, metav1.GetOptions{})
 	if err != nil {
 		klog.Errorf("No matching service for %v", svcExport)
-		return nil
+		return nil, true
```

Comment:
True will result in requeue. Set to false to retain existing behavior. As Tom said, this will be handled by #188 or #195

Reply:
I added this back since we Status will be distributed only in latest admiral. I will make it false.

Reply:
Ahh - ok - I noticed you had added it back and was going to comment...